apple/testing/default_runner/ios_xctestrun_runner.template.xctestrun

```diff
@@ -50,5 +48,35 @@ BAZEL_COMMAND_LINE_ARGS_SECTION
 BAZEL_SKIP_TEST_SECTION
 BAZEL_ONLY_TEST_SECTION
   </dict>
+  <key>__xctestrun_metadata__</key>
+  <dict>
+    <key>CodeCoverageBuildableInfos</key>
+    <array>
+      <dict>
+        <key>Architecture</key>
+        <string>BAZEL_ARCHITECTURE</string>
+        <key>BuildableIdentifier</key>
+        <string>000000000000000000000000:primary</string>
+        <key>IncludeInReport</key>
+        <false/>
+        <key>IsStatic</key>
+        <false/>
+        <key>Name</key>
+        <string>BAZEL_TEST_BUNDLE_NAME</string>
+        <key>ProductPath</key>
+        <string>FakePath</string>
```

Comment:
What is this normally? The XCTest bundle binary?

Reply:
I believe you're right. It is usually the path to the binary of the product for lack of a better term. For the app it would be `__TESTROOT__/Debug-iphonesimulator/Example.app/Example` and for the UI tests it would be the path to the xctest binary

Reply:
Let's make this correct, if we can. Seems like the xctest binary might be a fine thing to use since this is just the `CodeCoverageBuildableInfos` for the `.xctest` bundle.

Hosted app unit test:
```
				<key>ProductPath</key>
				<string>__TESTROOT__/Debug-iphonesimulator/iOSAppWithTests.app/PlugIns/iOSAppWithTestsTests.xctest/iOSAppWithTestsTests</string>
```

Non hosted app unit test:
```
				<key>ProductPath</key>
				<string>__TESTROOT__/Debug-iphonesimulator/iOSAppWithTestsTests-No-Host.xctest/iOSAppWithTestsTests-No-Host</string>
```

Framework unit test:
```
				<key>ProductPath</key>
				<string>__TESTROOT__/Debug-iphonesimulator/iOSFrameworkTests.xctest/iOSFrameworkTests</string>
```

UI test:
```
				<key>ProductPath</key>
				<string>__TESTROOT__/Debug-iphonesimulator/iOSAppWithTestsUITests-Runner.app/PlugIns/iOSAppWithTestsUITests.xctest/iOSAppWithTestsUITests</string>
```